core/src/main/java/me/uport/sdk/core/Networks.kt

```diff
@@ -8,17 +8,17 @@ object Networks {
 
     val mainnet = EthNetwork(
             "mainnet",
-            "0x01",
-            MNID.encode("0x01", "0xab5c8051b9a1df1aab0149f8b0630848b7ecabf6"),
+            "0x1",
```

Comment:
Why are we changing network IDs?

Reply:
The uPort app expects the shorthand encoding for network id's
The SDK supports both but it's simpler if the app receives what it expects.